src/main/java/core/basesyntax/service/parsefileinfo/CurrentStringParse.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.parsefileinfo;
+
+public class CurrentStringParse {
+    public String[] parse(String fileInfo) {
+        return fileInfo.split("[\\r\\n\\s]+");
+    }
+}
```

Comment:
I don`t think that whole separate class is required for string parsing, it could be a method in FruitParser